scripts/index.js

```diff
@@ -0,0 +1,42 @@
+function getRandom(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+let dieOne = {
+	value: 1,
+	roll: function() {
+		this.value = getRandom(1, 6);
+		return this.value;
+	}
+};
+
+let dieTwo = {
+	value: 1,
+	roll: function() {
+		this.value = getRandom(1, 6);
+		return this.value;
+	}
+};
+
+function printNumber(number1, number2) {
+
+	let value1 = document.querySelector('#dice1');
+	let value2 = document.querySelector('#dice2');
+	value1.style['background-image'] = `url(/images/inverted-dice-${number1}.svg)`;
+	value2.style['background-image'] = `url(/images/inverted-dice-${number2}.svg)`;
+}
+
+let button = document.querySelector('button');
+
+button.addEventListener('click', function(event) {
+
+	event.preventDefault();
+
+	let result1 = dieOne.roll();
```

Comment:
You don't actually have to store the return of the dice roll into a variable. You can also just roll the dice and then pass `dice.value` into the `printNumber()` function below, since the role method **actually** changes the `value` property on the dice object. Like this:

```javascript
dieOne.roll();
dieTwo.roll();

printDice(dieOne.value, dieTwo.value);
```